renovate.json

```diff
@@ -1,28 +1,13 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "platform": "github",
-  "labels": ["renovate"],
   "extends": [
-    "config:base",
-    ":rebaseStalePrs"
+    "github>gsoft-inc/renovate-config"
   ],
-  "enabledManagers": [
-    "github-actions",
-    "nuget"
-  ],
-  "stabilityDays": 3,
-  "prHourlyLimit": 0,
-  "prConcurrentLimit": 0,
-  "branchConcurrentLimit": 0,
-  "dependencyDashboard": false,
-  "gitAuthor": "Renovate Bot <bot@renovateapp.com>",
   "packageRules": [
     {
-      "matchManagers": ["nuget"],
-      "groupName": "NuGet dependencies"
-    },
-    {
-      "matchManagers": ["nuget"],
+      "matchManagers": [
+        "nuget"
+      ],
```

Comment:
- Xunit rules should be removed
- `Microsoft\\.CodeAnalysis` should only target the src files. In tests, you may want to use the latest.

Reply:
The rule XUnit was related to debug issue for Roslyn Rules was this fix?

Not sure to understand the comment for Microsoft\\.CodeAnalysis: only apply the exclusion when under the /src folder?

Reply:
- For analyzers, you need to update the code to use the right NuGet packages (we can do it together tomorrow if you want, or have a look at https://github.com/gsoft-inc/wl-extensions-mongo/commit/9ba066e4b3b5ec0847b5d82cd702ead4d8a4891f#diff-9ec05a9128808e24edbda1b22dc753726d4f8ca87c875e8771e796e43d17a8b4R14)
- I think we should update Microsoft.CodeAnalysis packages in tests projects. (sorry I use a single comment for all my feedbacks)
- Also, we should remove old dependencies such as `"^Microsoft\\.SourceLink\\.GitHub"`